From 867a546b98ead8e5e22061e3d460e71389260a01 Mon Sep 17 00:00:00 2001
From: Michael Bestas <mkbestas@lineageos.org>
Date: Tue, 16 Jan 2018 19:53:02 +0200
Subject: [PATCH 2/3] legacy: Add back legacy sensors rules

* Essentially reverts cef71b668db0170dc065ffca36e2a6d617b2bc62
* Required for msm8974 sensors.qcom

Change-Id: Ibff62ba7bb3a4c6a357f4c5e96496c75f281e0dd
---
 legacy-common/sensors.te | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/legacy-common/sensors.te b/legacy-common/sensors.te
index d6c701a..da15596 100644
--- a/legacy-common/sensors.te
+++ b/legacy-common/sensors.te
@@ -1,2 +1,9 @@
+type_transition sensors system_data_file:{ dir file } sensors_data_file;
+
+# Create directories and files under /data/misc/sensors
+# and /data/system/sensors.
+allow sensors sensors_data_file:dir create_dir_perms;
+allow sensors sensors_data_file:file create_file_perms;
+
 binder_use(sensors)
 binder_call(sensors, servicemanager)
-- 
2.7.4

